packages/server/postgres/migrations/1642094555266_addJiraServerIntegrationProvider.ts

```diff
@@ -0,0 +1,61 @@
+import {Client} from 'pg'
+import getPgConfig from '../getPgConfig'
+
+export async function up() {
+  const client = new Client(getPgConfig())
+  await client.connect()
+  await client.query(`
+  DO $$
+  BEGIN
+    ALTER TYPE "IntegrationProviderServiceEnum" ADD VALUE 'jiraServer';
+    ALTER TYPE "IntegrationProviderAuthStrategyEnum" ADD VALUE 'oauth1';
+    ALTER TABLE "IntegrationProvider"
+      ADD COLUMN IF NOT EXISTS "consumerKey" VARCHAR(255),
+      ADD COLUMN IF NOT EXISTS "privateKey" TEXT;
```

Comment:
+1 nit: I _think_ we could get away with a VARCHAR(1024) or VARCHAR(2048) because we know it's going to be an RSA-1024 (+ a little extra for base64 encoding maybe?). if it won't always be RSA1024 then TEXT is probably the right solution

Reply:
There is no benefit of using `VARCHAR(1024)` over `TEXT` except that the DB will catch malicious inserts maybe. According to Postgres recommendation I shouldn't make up arbitrary limits.
See Tip at https://www.postgresql.org/docs/9.1/datatype-character.html

Reply:
well defended! i like it